_overviews/scala3-scaladoc/blog.md

```diff
@@ -33,4 +33,29 @@ Scaladoc loads blog if the `_blog` directory exists.
 All the blogpost filenames should start with date in numeric format matching `YYYY-MM-DD`.
 Example name is `2015-10-23-dotty-compiler-bootstraps.md`.
 
+## Structure
 
+When creating a blog page, you will have a first part to fill in with different fields. See the following example:
+
+```
+---
+layout: <A reference to the layout page for the blog page>
+author: <Name of the author of the page>
+title: <Title of the page>
+subTitle: <Subtitle of the page>
+excerpt_separator: <!--more-->
+date: <Date of the creation of the page>
+authorImg: <Link to the author's image>
+---
+<Content>
+```
+
+## Special characters for the content
+
+` ```<content>``` ` : For code
```

Comment:
I'm specify that typically these are _code blocks_ which are meant to be multi-line and can include a language type.